CMakeLists.txt

```diff
@@ -42,7 +40,14 @@ if(",${CMAKE_SOURCE_DIR}," STREQUAL ",${CMAKE_BINARY_DIR},")
 	message(FATAL_ERROR "In-source builds are not supported.")
 endif()
 
-option(DEVELOPER_MODE "enable developer checks" OFF)
+option(PERFORM_STYLE_CHECKS
+	"check coding style, license headers (requires perl)" ON)
+option(BUILD_TESTS "build and enable tests" ON)
```

Comment:
What else would it be about?
Building tests, and what it implies: being able to run the tests.